### ISSUE TITLE
Source tree and go module versioning

### ISSUE THREAD

**Issue by sio4** [gobuffalo/packr#294]
It seems like v1.30.1 which was released on 25 Jun 2019 is the last v1 version of packr. If we don't update or fix version 1 nowadays, or even though we do it, I don't think we need to keep version 1 on the root directory and hold version 2 in a separated subdirectory in parallel. Can we create `v1` branch and clean up `master` for `v2` only? (on the root)

What do you think @paganotoni @stanislas-m?

**Comment by sio4** [comment]
I am testing this. https://github.com/gochigo/packr/tree/version2-only.

**Comment by sio4** [comment]
Could you please create a new branch `v1`? Then I will make a PR from https://github.com/gochigo/packr/tree/v1 and then will make an another PR from https://github.com/gochigo/packr/tree/version2-only against `master` so we can separate those two versions and mainly maintain v2 in the `master` while we keep using `packr` on buffalo and other projects.

**Comment by paganotoni** [comment]
Hey @sio4 thanks for putting this one together. I don't feel like I have the knowledge of Packr to vote on the proposal, I would let @stanislas-m weigh in. I truly appreciate the effort you're putting on this.

**Comment by sio4** [comment]
Sure! thank you @paganotoni for checking the proposal anyway.

Hi @stanislas-m, sorry for a series of review requests. I hope we can modernize all the dependencies and structures so we can make buffalo easier to maintain.
